openapi/components/schemas/GatewayAccountConfig/Adyen.yaml

```
@@ -68,6 +68,11 @@ allOf:
                     - Commission
                     - Remainder
                     - PaymentFee
+          salesTaxAmount:
+            type: number
+            format: double
+            description: Sales tax amount as a decimal to send with level 2/3 data.
```

suggestion: may be read as 2 thirds
```suggestion
            description: Sales tax amount as a decimal to send with level 2 or level 3 data.
```

---

```suggestion
            description: Sales tax amount as a decimal to send with level 2 and 3 enhanced scheme data. 
```